src/utils/wallets/getPrivateKey.js

```diff
@@ -30,7 +30,10 @@ function getPrivateKey(wallet: Wallet, internalKey: Uint8Array, encryptionType:
       !network ||
       !derivationPath
     ) {
-      throw new WalletInconsistentDataError(`Invalid mnemonic type ${type}`, wallet.id)
+      throw new WalletInconsistentDataError(
+        { walletId: wallet.id },
+        `Invalid mnemonic type ${type}`,
```

Comment:
`checkMnemonicType(type)` above checks `type === 'mnemonic'`
so message `Invalid mnemonic type ${type}` always will be _"Invalid mnemonic type **mnemonic**"_